www/class/config-generate-remote/relations/contactServicecommandsRelation.class.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * Copyright 2005 - 2019 Centreon (https://www.centreon.com/)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+namespace ConfigGenerateRemote;
+
+class contactServicecommandsRelation extends AbstractObject
```

Comment:
CamelCase only for attribute names and variables.
Please to add the PHP documentation for all methods.